src/main/menu.main.ts

```diff
@@ -365,6 +365,10 @@ export class MenuMain extends BaseMenu {
                 click: () => this.main.messagingService.send('lockVault'),
                 accelerator: 'CmdOrCtrl+L',
             },
+            {
+                label: this.i18nService.t('quitBitwarden'),
+                role: 'quit',
```

Comment:
This is going to result in two quit menu options on mac. See https://github.com/bitwarden/jslib/blob/master/src/electron/baseMenu.ts#L121

Reply:
@kspearrin Hm I see. But as a Windows/Linux user it would make more sense if the close button is located in the "File" menu I guess. What do you think?

Reply:
I don't have a problem with it being on the File menu for Windows and Linux. We just dont want it duplicated on macOS .

Reply:
@kspearrin I have now enabled the menu only for windows and linux